lib/spack/spack/test/cmd/ci.py

```diff
@@ -58,6 +59,7 @@ def set_env_var(key, val):
     os.environ[key] = val
 
 
+@pytest.mark.skipif(sys.platform == 'win32', reason="Error on Win")
```

Comment:
It looks like all tests in this module were skipped - I made a comment somewhere about how you could disable tests at the class/module level: can you do that here as well?

Reply:
In second review PR.